### PR TITLE
Feature/update for 3.0

### PR DESCRIPTION
**Pull request by faejr**

### CODE REVIEW

**Comment by helsingborgsstad** [comment]
Task linked: [CU-2z5mnrt TImeline fungerar inte i 3.0](https://app.clickup.com/t/2z5mnrt)

File: source/php/Module/views/timeline.blade.php [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<div class="{{ $classes }}" {!! $attributes !!}>
+    @if (!$hideTitle && !empty($post_title))
+        <h4>{!! apply_filters('the_title', $post_title) !!}</h4>
```
Comment (sebastianthulin) [inline source/php/Module/views/timeline.blade.php]: Use typography component.

File: source/sass/modularity-timeline.scss [hunk 2]
```diff
@@ -0,0 +1,2 @@
+.modularity-mod-timeline {
```
Comment (sebastianthulin) [inline source/sass/modularity-timeline.scss]: Not used? Remove.

File: source/php/Admin/Settings.php [hunk 3]
```diff
@@ -0,0 +1,28 @@
+<?php
```
Comment (sebastianthulin) [inline source/php/Admin/Settings.php]: Do we need a settings page for this addon?

**Pull request merged**